build.rs

```diff
@@ -822,3 +824,25 @@ where
         }
     }
 }
+
+fn lvi_mitigation_not_supported(base_config: &cc::Build) -> bool {
+    let feature_flag = base_config.is_flag_supported("-mlvi-hardening");
+
+    match feature_flag {
+        Ok(false) => true,
+        Err(_) => true,
+        _ => false,
+    }
+}
```

Comment:
```suggestion
fn lvi_mitigation_supported(base_config: &cc::Build) -> bool {
    base_config.is_flag_supported("-mlvi-hardening").unwrap_or(false)
}
```

Affirmative bool name sounds better.